tensorflow/tools/lib_package/BUILD

```diff
@@ -120,9 +119,8 @@ genrule(
         "@jpeg//:LICENSE.md",
         "@libxsmm_archive//:LICENSE",
         "@local_config_sycl//sycl:LICENSE.text",
-        "@nanopb_git//:LICENSE.txt",
```

Comment:
That gRPC change was committed. Can you:
- Update to the latest gRPC commit.
- Rebase with tensorflow master.
- And add the license back here

Thanks!